install_config/configuring_ephemeral.adoc

```diff
@@ -0,0 +1,62 @@
+[[install-config-configuring-ephemeral-storage]]
+= Configuring Ephemeral Storage
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+{product-title} can be configured to allow management of ephemeral
+storage of pod and container working data.  While pods and containers
```

Comment:
s/pod can container/container/

resources limits are currently applied only to containers

Reply:
+1